composer.json

```diff
@@ -21,6 +21,9 @@
         "phpunit/phpunit":          "^9.2.6",
         "vimeo/psalm":              "^3.12.2"
     },
+    "replace": {
+        "composer/package-versions-deprecated": "^1"
```

Comment:
This package has a different API from 1.x, so I'm not sure if we're actually replacing this properly

Reply:
I understand, but both have same-name classes, so they conflict anyway.

Reply:
Yes, but they don't have the same constants :D